pkgs/development/libraries/icu/make-icu.nix

```diff
@@ -93,7 +93,8 @@ let
   };
 
   buildRootOnlyAttrs = baseAttrs // {
-    name = pname + "-build-root-" + version;
+    pname = pname + "-build-root";
```

Comment:
checked that this indeed leads to `icu4c-build-root-74.2` for `icu.buildRootOnly`